jaeger-core/src/main/java/io/jaegertracing/internal/MDcScopeManager.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2018, The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.jaegertracing.internal;
+
+import io.opentracing.Scope;
+import io.opentracing.ScopeManager;
+import io.opentracing.Span;
+import io.opentracing.util.ThreadLocalScopeManager;
+
+public class MDcScopeManager implements ScopeManager {
```

Comment:
why the odd capitalization? It should be MDCScopeManager (MdcScopeManager is also reasonable but not as good).

Reply:
when I use MDC the CI/CD fails with the following error:

Abbreviation in name 'MDCScope' must contain no more than '2' consecutive capital letters.